sse2neon.h

```diff
@@ -575,6 +580,21 @@ FORCE_INLINE __m128 _mm_loadu_ps(const float *p)
     return vreinterpretq_m128_f32(vld1q_f32(p));
 }
 
+// Loads a double-precision, floating-point value.
+// The upper double-precision, floating-point is set to zero. The address p does
+// not need to be 16-byte aligned.
+// https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2010/574w9fdd(v%3dvs.100)
+FORCE_INLINE __m128d _mm_load_sd(const double *p)
+{
+#if defined(__aarch64__)
+    return vsetq_lane_f64(*p, vdupq_n_f64(0), 0);
+#else
+    const float *fp = (const float *)p;
```

Comment:
Use `clang-format` to ensure consistent style.

Reply:
Done.